### ISSUE TITLE
1.5.1 issues

### ISSUE THREAD

**Issue by langdonx** [leftstick/vscode-esformatter#4]
With the release of 1.5.1, I've run into 2 issues:

**Empty functions get a space put into their body:**
    1. `const a=function(){};` -> `const a = function () { };`
    2. `const b=()=>{};` -> `const b = () => { };`

1.5.1 actually would force a break on the empty arrow functions ([fixed in esformatter](https://github.com/millermedeiros/esformatter/pull/488), but sadly not released).

The **`whiteSpace.after.FunctionReservedWord: 0` setting is seemingly ignored:**
You can see it in the above example, but: `go(function(){});` -> `go(function () { });`

I'm assuming these are because of `esformatter-curly`?  Are there options available to stop the extension from using this plugin?

**Comment by leftstick** [comment]
For what plugins you want to activate, just change your `.esformatter`, see [mine](https://github.com/leftstick/vscode-esformatter/blob/master/.esformatter#L446-L452)

I have 5 plugins activated:

```javascript
{
  "plugins": [
    "esformatter-braces",
    "esformatter-collapse-objects",
    "esformatter-dot-notation",
    "esformatter-literal-notation",
    "esformatter-quotes"
  ]
}
```

>No built-in plugins will be activated automatically

>for `esformatter` issue, i will keep following up

**Comment by langdonx** [comment]
It doesn't appear that the extension is formatting my JavaScript files anymore 1.5.1.  The results I was seeing appear to be from VS Code's default JavaScript formatter.  I get the same results if I uninstall `vscode-esformatter`. 

Is there a log somewhere or something I can do to figure out why the extension isn't being used when I format document?  The Output tab is empty when I select Extensions.

**Comment by langdonx** [comment]
Ah, this appears to be the issue.  I saw those dependencies were updated.  Any ideas?

![image](https://user-images.githubusercontent.com/441982/27941672-c79292dc-629f-11e7-97c2-e435640c373c.png)

**Comment by leftstick** [comment]
Seems the dependency `vscode-languageclient` was not packaged along with the extension. 
I even tried rollback to `1.5.0`, and package the extension with latest `vsce`(official tool for publishing), have the same issue either. Therefor, i think the issue might be caused the the new publishing tool.

Let me report it to `vscode` team to see if they have any idea about it. Otherwise, i will refactor the extension in another way

**Comment by leftstick** [comment]
@langdonx issue fixed, Please give a try with `1.5.2`

**Comment by langdonx** [comment]
Looking good, thank you!
